actions-server/src/handlers/joinGame.ts

```diff
@@ -70,6 +72,24 @@ const handler = async (req: Request, res: Response) => {
       process.env.HASURA_GRAPHQL_JWT_SECRET || "missing secret"
     )
 
+    if (!hostExists) {
+      const { errors } = await graphQLClient({ jwt: token }).BecomeHost({
+        gameId,
+        playerId,
+      })
+      if (errors) {
+        console.log(
+          `Player (id: ${playerId}, client_uuid: ${clientUuid}) in game (id: ${gameId}) failed to become the HOST; ${errors.join(
+            ","
+          )}`
+        )
```

Comment:
was hesitant to throw an error here, to make sure the JWT still gets issued regardless of whether they became the host or not (is there possibility of a race condition?)